core/services/chainlink/testdata/dump/evm-db.json

```diff
@@ -0,0 +1,95 @@
+{
+  "EVMChains": [
+    {
+      "ID": "0",
+      "CFG": {
+        "Enabled": true,
+        "BlockHistoryEstimatorBlockDelay": "99",
+        "BlockHistoryEstimatorBlockHistorySize": "99",
+        "BlockHistoryEstimatorEIP1559FeeCapBufferBlocks": "99",
+        "ChainType": "foobar",
+        "EthTxReaperThreshold": "10m",
+        "EthTxResendAfterThreshold": "5m10s",
+        "EvmEIP1559DynamicFees": true,
+        "EvmFinalityDepth": "99",
+        "EvmGasBumpPercent": "99",
+        "EvmGasBumpTxDepth": "99",
+        "EvmGasBumpWei": "159",
+        "EvmGasFeeCapDefault": "9876",
+        "EvmGasLimitDefault": "99",
+        "EvmGasLimitMultiplier": 1.75,
+        "EvmGasPriceDefault": "3456",
+        "EvmGasTipCapDefault": "4321",
+        "EvmGasTipCapMinimum": "7",
+        "EvmHeadTrackerHistoryDepth": "99",
+        "EvmHeadTrackerMaxBufferSize": "99",
+        "EvmHeadTrackerSamplingInterval": "1h7s",
+        "EvmLogBackfillBatchSize": "99",
+        "EvmLogPollInterval": "43s",
+        "EvmMaxGasPriceWei": "9000000",
+        "EvmNonceAutoSync": false,
+        "EvmUseForwarders": false,
+        "EvmRPCDefaultBatchSize": "99",
+        "FlagsContractAddress": "0xae4E781a6218A8031764928E88d457937A954fC3",
+        "GasEstimatorMode": "FixedPrice",
+        "KeySpecific": null,
```

Comment:
I should add one more case to cover `KeySpecific`...
- [ ] `KeySpecific` test case